src/createList.js

```diff
@@ -0,0 +1,50 @@
+import './style.css';
+
+const listContainer = document.querySelector('.list-container');
+// eslint-disable-next-line import/no-mutable-exports
+export let lists = [];
+// eslint-disable-next-line no-unused-vars
+let index = 1;
+
+// local storage
+export const saveData = () => {
+  localStorage.setItem('dataList', JSON.stringify(lists));
+};
+
```

Comment:
- [ ] Editing a list is not working as intended. When a task _one_ is added and then then renamed to _one man_, the changes do not reflect in the local storage. Kindly endeavor to improve upon this aspect.

**Editing / Update error:**
![image](https://user-images.githubusercontent.com/12745474/149446265-cb934706-f5c7-4ab2-95ac-9d5c071a327b.png)